### PR TITLE
Document Registration

### PR DESCRIPTION
**Pull request by gregorydlogan**
This PR adds documentation regarding why we want adopters to register, and exactly what they are sending if they do choose to.

### Your pull request should…

* [x] have a concise title
* [ ] ~[close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists~
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

**Comment by oas777** [comment]
So... where does that documentation live? Adopters Guide?

**Comment by gregorydlogan** [comment]
This would land in https://docs.opencast.org/r/13.x/admin/#registration/, and get merged forward from there into the newer branches.

**Comment by oas777** [comment]
Perfect, thanks.

File: docs/guides/admin/docs/registration.md
Comment (ziegenberg) [inline docs/guides/admin/docs/registration.md:1]: Hi @gregorydlogan! Out of curiosity: Is there a reason for typing two spaces after a period? It doesn't matter for markdown or HTML anyway; I just wanted to know why.

File: docs/guides/admin/docs/registration.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+The statistical data we gather includes:
+
+* The number of recordings, and series
+* The number of capture agents
+* The number of users
+* The number of Opencast jobs
+* The sum of the duration of your recordings
+* The number of tenants
+* The number of Opencast nodes, including their configurations
```
Comment (ziegenberg) [inline docs/guides/admin/docs/registration.md]: How is "including their configuration" defined? We do not upload the config files, right? In any case, this should be specified.

File: docs/guides/admin/docs/registration.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+* The version of Opencast in use
+* Tobira's statistical data, if any
```
Comment (ziegenberg) [inline docs/guides/admin/docs/registration.md:32]: How is "Tobira's statistical data" defined?

File: docs/guides/admin/docs/registration.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+Registering your Opencast installation
+======================================
+
+Why we want you to register
+---------------------------
+
+Measuring the health, and uptake of an Open Source project can be difficult at the best of times, especially when you
+consider the privacy needs of your adopters. Registering your Opencast installation is entirely optional, but it is the
```
Comment (ziegenberg) [inline docs/guides/admin/docs/registration.md]:
```suggestion
Measuring the health, and uptake of an Open Source project can be difficult at the best of times, especially when considering your adopters' privacy needs.  Registering your Opencast installation is entirely optional, but it is the
```

File: docs/guides/admin/docs/registration.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+only way to appear on [the community map](https://map.opencast.org).  It is also the easiest, and best way to provide
+Opencast with up-to-date data about who your organiation is, and how they are using Opencast.  This information enables
```
Comment (ziegenberg) [inline docs/guides/admin/docs/registration.md]:
```suggestion
Opencast with up-to-date data about who your organisation is, and how they are using Opencast.  This information enables
```

File: docs/guides/admin/docs/registration.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+us to better focus our resources, and lets us track broad-stroke usage patterns so we can make Opencast better.
+
+What we gather
+--------------
+
+We gather two different classes of information: Biographical, and Statistical.  To register, you only need to provide
+biographical data, however statistical data is strongly recommended as well.  All of the fields we gather are optional,
```
Comment (ziegenberg) [inline docs/guides/admin/docs/registration.md]:
```suggestion
biographical data, however, statistical data is strongly recommended as well.  All of the fields we gather are optional,
```

File: docs/guides/admin/docs/registration.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+* The number of capture agents
+* The number of users
+* The number of Opencast jobs
+* The sum of the duration of your recordings
+* The number of tenants
+* The number of Opencast nodes, including their configurations
+* The version of Opencast in use
+* Tobira's statistical data, if any
+
+The exact data sent to our systems is presented to your prior to actually registering, so if you spot something in that
```
Comment (ziegenberg) [inline docs/guides/admin/docs/registration.md]:
```suggestion
The exact data sent to our systems is presented to you prior to actually registering, so if you spot something in that
```

**Review by ziegenberg — CHANGES_REQUESTED** [review]
I ran the text through Grammarly and it detected some spelling mistakes.

File: docs/guides/admin/docs/registration.md
Comment (gregorydlogan) [reply docs/guides/admin/docs/registration.md:1]: Force of habit.  That's how I learned, and (afaik) still how they're teaching it in English still.

File: docs/guides/admin/docs/registration.md (hunk 1 above)
Comment (gregorydlogan) [reply docs/guides/admin/docs/registration.md]: Good point!  I meant hardware configs (ram, cores, etc), but it's very unclear here.

File: docs/guides/admin/docs/registration.md (hunk 2 above)
Comment (gregorydlogan) [reply docs/guides/admin/docs/registration.md:32]: That depends on what Tobira sends.  Opencast just acts as a passthrough of a json blob - we don't process it at all.  The registration server gathers [these](https://github.com/opencast/adopter-registration-server/blob/master/app/models.py#L125) fields.  I can enumerate everything here, but that seems excessive.  What about something like `The number of realms, and blocks in Tobira, along with some other feature-use flags`?

Comment (ziegenberg) [reply docs/guides/admin/docs/registration.md:32]: That sounds good.

**Pull request merged**